DiscordChatExporter.Core/Exporting/Filtering/MessageFilter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using DiscordChatExporter.Core.Discord.Data;
+using DiscordChatExporter.Core.Exporting.Filtering.Parsing;
+using Superpower;
+using Superpower.Model;
+
+namespace DiscordChatExporter.Core.Exporting.Filtering
+{
+    public abstract partial class MessageFilter
+    {
+        public abstract bool Filter(Message message);
+    }
+
+    public partial class MessageFilter
+    {
+        public static MessageFilter CreateFilter(string key, string value)
+        {
+            return key.ToLower() switch
+            {
+                "contains" => new ContainsMessageFilter(value),
```

Comment:
This could probably be dropped and replaced with an overload of MessageFilter.CreateFilter that only takes one parameter if we don't want the redundant `contrains:` filter type.

Reply:
Yeah, I think we should probably drop the redundant `contains:` to avoid confusion